### DISCUSSION TITLE
Buds Pro Firmware Update?

### DISCUSSION THREAD

**Discussion by vinomaker73** [timschneeb/GalaxyBudsClient#184] [Q&A] [answered]
Can I use this program to update the Firmware on the Buds Pro?  I know there is an update available for them but don't know how to update them with this tool.

Thanks!

**Comment by timschneeb** [comment]
Update: I have added a firmware updater for the Buds+.
When I receive my new Buds Pro in the mail sometime next week, I'll also add support for them as well.

**Comment by timschneeb** [comment, marked as answer]
The latest version supports firmware updates for the Buds Pro and Buds+.
